src/Digdir.Domain.Dialogporten.WebApi/appsettings.Development.json

```diff
@@ -62,9 +61,9 @@
   "LocalDevelopment": {
     "UseLocalDevelopmentUser": true,
     "UseLocalDevelopmentResourceRegister": true,
-    "UseLocalDevelopmentAltinnAuthorization": true,
+    "UseLocalDevelopmentAltinnAuthorization": false,
```

Comment:
Reverter denne? 👻
(Og den under)

Reply:
Glemte selvfølgelig den, duh. La også tilbake Subscription-key igjen, vi kommer til å trenge den for navneoppslaget.